src/content/docs/index.mdx

```diff
@@ -1,25 +1,24 @@
 ---
-title: Welcome to Astro tips
-description: "TODO: tagline"
+title: Welcome to Astro tips!
+description: The place for content that isn't suitable for the official docs, for all Astronauts!
+tableOfContents: false
 ---
 
 import { Card, CardGrid } from '@astrojs/starlight/components';
 
-TODO: tagline
+The place for content that isn't suitable for the official docs, for all Astronauts!
 
-## Next steps
-
-<CardGrid stagger>
-	<Card title="Update content" icon="pencil">
-		Edit `src/content/docs/index.mdx` to see this page change.
+<CardGrid>
+	<Card title="Find community resources" icon="magnifier">
+		No more searching through blogs, it's centralized here! With credits of course.
 	</Card>
-	<Card title="Add new content" icon="add-document">
-		Add Markdown or MDX files to `src/content/docs` to create new pages.
+	<Card title="Add your own content" icon="pencil">
+		Found something interesting we don't have already? We want to know about it!
 	</Card>
-	<Card title="Configure your site" icon="setting">
-		Edit your `sidebar` and other config in `astro.config.mjs`.
+	<Card title="Share and learn from others" icon="open-book">
+		Learn new tips and tricks about Astro and share any you pick up!
 	</Card>
-	<Card title="Read the docs" icon="open-book">
-		Learn more in [the Starlight Docs](https://starlight.astro.build/).
+	<Card title="Help people discover gems" icon="information">
+		Lorem ipsum
```

Comment:
Lorem ipsum?

Reply:
yeah i know! we'll see tomorrow during the call